### PR TITLE
Enabled view scope pre destroy test faces40

### LINKED ISSUE #24546: Enable testViewScopedJSFManagedBeanPreDestroy for EE10

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #24546

### CODE REVIEW

**Comment by pnicolucci** [comment]
#libby #build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.2_fat

**Comment by pnicolucci** [comment]
#libby #build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.2_fat

**Comment by pnicolucci** [comment]
#libby #build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.2_fat

**Review by volosied — APPROVED** [review]
Looks fine to me.

File: dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22AparTests.java [hunk 1]
```diff
@@ -507,7 +503,7 @@ public void testViewScopedJSFManagedBeanPreDestroy() throws Exception {
     // Skipping for EE10 as this test uses both a CDI bean and a ManagedBean and ManagedBeans are
     // no longer available in Faces 4.0. The testViewScopedJSFManagedBeanPreDestroy test will
     // be executed for EE10 with just a CDI bean to verify that PreDestroy is called.
-    @SkipForRepeat(EE10_FEATURES) // Skipped due to HTMLUnit / JavaScript Incompatabilty (New JS in RC5)
```
Comment (volosied) [inline dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22AparTests.java:510]: Sorry for these misplaced comments.  I used find / replace all and shouldn't have...

**Pull request merged**